services/jetbrains/jetbrains.tester.js

```diff
@@ -121,7 +142,20 @@ t.create('empty XML')
       'Content-Type': 'text/xml;charset=UTF-8',
     }
   )
-  .expectJSON({ name: 'downloads', value: 'invalid' })
+  .expectJSON({ name: 'downloads', value: 'unparseable xml response' })
+
+t.create('XML with unknown root')
+  .get('/plugin/d/9435.json')
+  .intercept(
+    nock =>
+      nock('https://plugins.jetbrains.com')
+        .get('/plugins/list?pluginId=9435')
+        .reply(200, '<?xml version="1.0" encoding="UTF-8"?><plugin />'),
+    {
+      'Content-Type': 'text/xml;charset=UTF-8',
+    }
+  )
+  .expectJSON({ name: 'downloads', value: 'invalid response data' })
 
 t.create('404 status code')
```

Comment:
This test (and probably others) are no longer relevant as they're now covered by the parent classes.

Reply:
I agree that the parent class is responsible for checking response data against a schema and this is tested somewhere. But with all tests with `value: 'invalid response data'` I want to verify that schema is defined properly, e.g. `required()` set on required elements ;-) 
If I remove one of the `required()` in the schema this test will fail with `TypeError: Cannot read property 'category' of undefined`
I would like to test schema. Of course it can be done in another way. What do you think about this?